test/inductor/test_custom_partitioner_fn.py

```diff
@@ -0,0 +1,71 @@
+# Owner(s): ["module: inductor"]
```

Comment:
nit: this can be `"module: pt2-dispatcher"` (since partitioner is owned by pt2-dispatcher, although I agree it's confusing since inductor is the one that calls it 😛 )